src/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java

```diff
@@ -94,6 +94,7 @@ public static void setupResearches() {
 	    Slimefun.registerResearch(new Research(89, "Night Vision Goggles", 10), SlimefunItems.NIGHT_VISION_GOGGLES);
 	    Slimefun.registerResearch(new Research(90, "Pickaxe of Containment", 14), SlimefunItems.PICKAXE_OF_CONTAINMENT, SlimefunItems.BROKEN_SPAWNER);
 	    Slimefun.registerResearch(new Research(91, "Hercules Pickaxe", 28), SlimefunItems.HERCULES_PICKAXE);
+	    Slimefun.registerResearch(new Research(92, "Saw Mill", 2), SlimefunItems.SAW_MILL);
 	    Slimefun.registerResearch(new Research(92, "Table Saw", 2), SlimefunItems.TABLE_SAW);
```

Comment:
Now you registered the same Research twice. Just add the two Items to the same Research:

Slimefun.registerResearch(new Research(92, "Table Saw", 2), SlimefunItems.SAW_MILL, SlimefunItems.TABLE_SAW);

Reply:
Ah, I see